tools/postinstall/manifest_externs_hermeticity.patch

```diff
@@ -1,11 +1,11 @@
 diff --git node_modules/@bazel/typescript/internal/build_defs.bzl node_modules/@bazel/typescript/internal/build_defs.bzl
-index 7fef44c..4e25ed7 100644
+index 03cb68fdf..2f604234c 100755
 --- node_modules/@bazel/typescript/internal/build_defs.bzl
 +++ node_modules/@bazel/typescript/internal/build_defs.bzl
-@@ -93,25 +93,23 @@ def _filter_ts_inputs(all_inputs):
-         if f.path.endswith(".js") or f.path.endswith(".ts") or f.path.endswith(".json")
+@@ -102,17 +102,17 @@ def _filter_ts_inputs(all_inputs):
+         if f.extension in ["js", "jsx", "ts", "tsx", "json"]
```

Comment:
What's the reason for this change?

Reply:
no reason, probably I made something when creating the git diff patch

Reply:
sorry, i was checking this one, but the reason is that build_defs has changes in version 1.6.0 and the post install script failed to apply the patch (the 2nd hunk)
https://github.com/bazelbuild/rules_nodejs/blob/bd83907954d1b3cd53dff01c37e3c4964890ef5c/packages/typescript/src/internal/build_defs.bzl

Reply:
I see. makes sense. I thought it was part of our patch, but it's obviously just needed an update of the base for getting the diff.